api/server.py

```diff
@@ -101,7 +101,11 @@ def equations_to_amr(
 
 @app.post("/code_to_amr")
 def code_to_amr(
-    code_id: str, name: Optional[str] = None, description: Optional[str] = None, redis=Depends(get_redis)
+    code_id: str,
+    name: Optional[str] = None,
+    description: Optional[str] = None,
+    dynamics_only: Optional[bool] = False,
```

Comment:
Can we add this param to the doc string(s)?

Reply:
Yep let me do that!